src/services/user-service.tsx

```diff
@@ -45,20 +44,21 @@ export const userService = {
       })
     })
   },
-
-  updateUser: (
-    userId: string,
-    params: UpdateUserParams
-  ): Promise<AxiosResponse<null>> => {
-    return axiosClient.patch(createUrlPath(URLs.users.update, userId), params)
+  updateUser: (userId: string, params: UpdateUserParams) => {
+    return baseService.request<null>({
+      method: 'PATCH',
+      url: getFullUrl({
+        pathname: `${URLs.users.update}/${userId}`
```

Comment:
```suggestion
        pathname: URLs.users.update,
        parameters: { id: userId }
```
and change ```update``` URL in ```request.ts``` file to: '/users/:id'

Can we make so?

Reply:
I tried this, but it will not work correct, because `update` endpoint don't have an `id`.

Reply:
We can add /:id to our user request – this will solve the issue since the user is updated only by ID. That way, we don’t need a separate /users/ request. I’d avoid concatenation here if possible, but up to you.

Reply:
Thanks) I will try to change it in the next issue.